15 - LocalStorage/index-START.html

```diff
@@ -26,13 +26,58 @@ <h2>LOCAL TAPAS</h2>
   </div>
 
 <script>
-  const addItems = document.querySelector('.add-items');
-  const itemsList = document.querySelector('.plates');
-  const items = [];
+  class customStoredList {
+    constructor(wrapperClass) {
+      this.wrapper = document.querySelector(wrapperClass);
+      this.addItems = this.wrapper.querySelector('.add-items');
+      this.itemsList = this.wrapper.querySelector('.plates');
+      this.savedItems = localStorage.getItem('items');
+      this.items = this.savedItems ? JSON.parse(this.savedItems) : [];
 
-</script>
+      this.renderList(this.items, this.itemsList);
```

Comment:
i would probably spawn the action as the last thing in the constructor. 1. assign variables 2. subscribe to events 3. do stuff

Reply:
makes sense. thx!